src/npcmove.cpp

```diff
@@ -1073,11 +1073,7 @@ bool wants_to_reload( const npc &who, const item &it )
 
 bool wants_to_reload_with( const item &weap, const item &ammo )
 {
-    if( ammo.is_magazine() && ( ammo.ammo_remaining() <= weap.ammo_remaining() ) ) {
-        return false;
-    }
-
-    return true;
+    return !( ammo.is_magazine() && ( ammo.ammo_remaining() <= weap.ammo_remaining() ) );
```

Comment:
Can be made even simpler:
```suggestion
    return !ammo.is_magazine() || ammo.ammo_remaining() > weap.ammo_remaining();
```

Reply:
Thanks!